packages/core/css-utils/src/getMediaQuery.ts

```diff
@@ -0,0 +1,11 @@
+import { CSSObject } from '@emotion/react';
+
+export const getMediaQuery = (mediaQuery: string, cssObject: CSSObject) => {
+  return {
+    [`@media (${mediaQuery})`]: cssObject,
+  };
+};
+
+export const mobileMediaQuery = (cssObject: CSSObject) => {
+  return getMediaQuery('max-width : 320px', cssObject);
+};
```

Comment:
```suggestion
export const mobileMediaQuery = (cssObject: CSSObject) => {
   return {
    '@media ('max-width: '320px')': cssObject,
  };
};
```

(^ 이거 약간 문법 오류가 있을 수도 잇어요!)

굳이 한번 래핑해서 할 필요를 모르겠어요! 요렇게 간단하게 하는 건 어떨까요?

Reply:
아 다른 곳에서 미디어 쿼리를 사용하는 곳이 있어서 아예 utils 함수로 만들어버렸는데 그냥 320만 적용이 되게 변경을 할까요?

Reply:
오 넹 만약에 뭐 테블릿 미디어 쿼리가 필요하면 똑같이 선언해도 그닥 반복된 코드가 많진 않아서 무관할 거 같아요

무작정 공통으로 다 묶어버리기 보다는 우선은 단순하게 가는게 좋을 거 같습니당